Data/Yaml/Pretty.hs

```diff
@@ -29,30 +31,47 @@ import qualified Data.Vector as V
 import Data.Yaml.Builder
 
 -- |
--- Since 0.8.13
+-- @since 0.8.13
 data Config = Config
   { confCompare :: Text -> Text -> Ordering -- ^ Function used to sort keys in objects
+  , confDropNull :: Bool
   }
 
--- | The default configuration: do not sort objects.
+-- | The default configuration: do not sort objects or drop keys
 --
--- Since 0.8.13
+-- @since 0.8.13
 defConfig :: Config
-defConfig = Config mempty
+defConfig = Config mempty False
 
 -- |
--- Since 0.8.13
+-- @since 0.8.13
 getConfCompare :: Config -> Text -> Text -> Ordering
 getConfCompare = confCompare
 
--- |
--- Since 0.8.13
+-- | Sets ordering for object keys
+--
+-- @since 0.8.13
 setConfCompare :: (Text -> Text -> Ordering) -> Config -> Config
 setConfCompare cmp c = c { confCompare = cmp }
 
+-- |
+-- @since 0.8.24
```

Comment:
Looks great! Could you also update the `.cabal` file with the new version and add a note to the changelog?

Reply:
Thanks for the quick review! I have bumped the version number.

Reply:
Awesome! Once Travis approves I'll merge this in. Thanks!